vaccine_feed_ingest/utils/normalize.py

```diff
@@ -5,6 +5,28 @@
 from typing import Optional, Tuple
 
 
+def organization_from_name(name: str) -> Optional[Tuple[str, str]]:
```

Comment:
I was trying to do all this work in `provider_id_from_name` by spiffing up the regex's, but there were no tests for `provider_id_from_name` so I felt a bit uneasy changing the regexes there.

Maybe later we can combine `organization_from_name` and `provider_id_from_name`?